New npm version asks not to "cache clean"

## Summary

Attempting to install this package with node 5.x will not work since it has a new self-checking mechanism (see npm output below)

## System information

 - Operating system: Ubuntu Xenial
 - Version: 0.17.x
- npm: 5.4.1
 - Browser: - Not applicable

## Output
npm ERR! As of npm@5, the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid. If you want to make sure everything is consistent, use 'npm cache verify' instead.
npm ERR! 
npm ERR! If you're sure you want to delete the entire cache, rerun this command with --force.

## How to reproduce

On regular install
sudo make install

## Solution proposed
As npm itself recommends, use the command "npm cache verify" in the Makefile. Using "npm cache clean --force" is not recommended and throws the proper warning to the CLI. I did it and package works just fine. 

## Real-life consequences (anything community should be aware, for instance how it affects your deployment)

Regular installing will throw an error and stop

--- a/Makefile
+++ b/Makefile
@@ -46,7 +46,9 @@ clean-test:
 	rm -fr htmlcov/
 
 clean-assets:
-	npm cache clean
+	# This is only relevant for npm<5
+	# https://github.com/learningequality/ka-lite/issues/5519
+	# npm cache clean
 	rm -rf kalite/database/templates/
 	rm -rf .kalite_dist_tmp
 

--- a/docs/installguide/release_notes.rst
+++ b/docs/installguide/release_notes.rst
@@ -8,6 +8,16 @@ to read the release notes.
   upgrading from ``0.16.x`` to ``0.17.x`` is fine - but upgrading from
   ``0.15.x`` to ``0.17.x`` is not guaranteed to work.
 
+
+0.17.4 (unreleased)
+-------------------
+
+Developers
+^^^^^^^^^^
+
+ * Do not use `npm clean`, now requires npm>=5 for building on unclean systems :url-issue:`5519`
+
+
 0.17.3
 ------
 

--- a/kalite/version.py
+++ b/kalite/version.py
@@ -3,7 +3,7 @@
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
 MAJOR_VERSION = "0"
 MINOR_VERSION = "17"
-PATCH_VERSION = "3"
+PATCH_VERSION = "4dev0"
 VERSION = "%s.%s.%s" % (MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION)
 SHORTVERSION = "%s.%s" % (MAJOR_VERSION, MINOR_VERSION)
 

